packages/viewer/src/components/viewer-dom-element/viewer-dom-element.tsx

```diff
@@ -137,6 +146,16 @@ export class ViewerDomElement implements HTMLDomRendererPositionableElement {
     this.syncScale();
   }
 
+  @Watch('matrix')
+  protected handleMatrixChanged(
```

Comment:
Please add an `@ignore` comment. Otherwise it shows up unnecessarily in our docs.